helper-cli/src/main/kotlin/commands/ScanStorageCommand.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+
+import org.ossreviewtoolkit.helper.commands.scanstorage.DeleteCommand
+
+class ScanStorageCommand : CliktCommand(
+    help = "Commands for working with scan storages."
+) {
+    init {
+        subcommands(
+            DeleteCommand()
+        )
+    }
+
+    override fun run() {
+        // Intentionally empty, because all logic is implemented inside the sub commands.
+    }
```

Comment:
You might want to derive from `NoOpCliktCommand` then instead.

Reply:
I see other helper-cli commands do the same. So maybe add another commit to address this for all of them?